examples/plot_ransac.py

```diff
@@ -65,10 +65,10 @@
 
 ###############################################################################
 # We import ``Ransac`` and run the familiar ``fit_transform`` method.
-from autoreject import Ransac
-from autoreject.utils import interpolate_bads
+from autoreject import Ransac  # noqa
+from autoreject.utils import interpolate_bads  # noqa
 
-ransac = Ransac(verbose='tqdm')
+ransac = Ransac(verbose='progressbar', n_jobs=1)
```

Comment:
It's not a hard dependency. It was just in the example to illustrate ... but okay for changing it to `'progressbar'`